gammapy/data/tests/test_observations.py

```diff
@@ -228,6 +228,15 @@ def test_observation():
     assert_allclose(obs.muoneff, 1)
 
 
+@requires_data()
+def test_observation_read():
```

Comment:
There is no test for the case of a separate irf file

Reply:
Added now